pkg/logger/worker.go

```diff
@@ -85,6 +90,26 @@ func (w *Worker) sendCloudEvent(logReq LogRequest) error {
 		return fmt.Errorf("while creating http transport: %w", err)
 	}
 
+	if logReq.Url.Scheme == "https" {
+		caCertFile, err := os.ReadFile(filepath.Join(CaCertPath, logReq.TlsCertName))
+		// Do not fail if certificates not found, for backwards compatibility
```

Comment:
maybe log a warning might be good, missing the cert might be not intentional.

Reply:
Added log warning for the case when logger endpoint is https but CA cert file not found